build.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0"?>
+
+<project name="PhingGitHub example" default="create-release-assets" basedir="." description="This is an example buildfile for GitHub Phing tasks.">
+    
+    <!-- Import the github phing tasks. -->
+    <import file="${project.basedir}/import.xml" />
+
+    <if>
+        <available file="${project.basedir}/build.props.local" />
+        <then>
+            <property file="${project.basedir}/build.props.local" />
```

Comment:
Idea: How about supporting loading authentication information from environment variables as well as a properties file? 

[Other tools](https://github.com/github/hub/issues/245) support `GITHUB_USER`, `GITHUB_PASSWORD` etc.

The [`PropertyTask`](https://github.com/github/hub/issues/245) seems to make this quite easy. It will probably require renaming of the properties in `build.props.local`.